### PR TITLE
Update trace text format

### PR DESCRIPTION
**Pull request by mankinchi**
Handling the styling issue mentioned in https://github.com/NetrunnerDB/netrunnerdb/issues/623. This would also allows some cards data change to work properly afterward

### CODE REVIEW

**Review by plural — CHANGES_REQUESTED** [review]
Please update to keep the match against the trace in the json as well.

Please also run locally and make sure things look ok.

File: src/AppBundle/Service/CardsData.php [hunk 1]
```diff
@@ -888,7 +888,7 @@ public function getCardInfo(Card $card, string $locale)
         // If the card has text
         if (strlen($cardinfo['text']) > 0) {
             // replacing <trace>
-            $cardinfo['text'] = preg_replace('/<trace>([^<]+) ([X\d]+)<\/trace>/', '<strong>\1 [\2]</strong>–', $cardinfo['text']);
+            $cardinfo['text'] = preg_replace('/<trace>.*([X\d]+)<\/trace>/', '<span>Trace [\1]</span>.', $cardinfo['text']);
```
Comment (plural) [inline src/AppBundle/Service/CardsData.php:891]: please make the `.*` not greedy with `.*?` or use the original capture group here.

**Review by plural — CHANGES_REQUESTED** [review]
just one small thing.

**Comment by mankinchi** [comment]
so sorry for the disappearance. Personal issue is annoying but resolved. I will look back into this and update it asap 😄

**Comment by plural** [comment]
I ended up taking care of this in #725 .  Thank you for kicking this off!

**Pull request closed without merging**